recipe/meta.yaml

```diff
@@ -66,30 +65,30 @@ requirements:
     - gobject-introspection 1.*
     - glib {{ glib }}
     - gettext 0.21.0 # [osx]
-    - harfbuzz {{ harfbuzz }}
-    - fontconfig {{ fontconfig }}
-    - freetype {{ freetype }}
-    - libiconv  # [win and osx]
+    - harfbuzz {{ harfbuzz }}      # [not win]
+    - fontconfig {{ fontconfig }}  # [not win]
+    - freetype {{ freetype }}      # [not win]
+    - libiconv 1.16                # [osx]
```

Comment:
`pango 1.50` is mentioned in `rsvg/Cargo.toml` but I can't see any reference to `libiconv`.  Where is its pinning coming from?

Reply:
I removed it. Thanks

Reply:
Well, not that easy. It fails `osx` without `libiconv`:
```
  ERROR (librsvg,bin/rsvg-convert): Needed DSO lib/libiconv.2.dylib found in ['defaults/osx-arm64::libiconv==1.16=h80987f9_3']

  ERROR (librsvg,bin/rsvg-convert): .. but ['defaults/osx-arm64::libiconv==1.16=h80987f9_3'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
```

Reply:
I returned it. I'm not sure why it's happening. Is it glib, fontconfig, or libxml2 can cause it?

Reply:
Looks like an extra `libiconv` on `osx-64` for `libxml2` (at least):
```
$CONDA_SUBDIR=osx-64 conda search libxml2=2.13.5 --info
...
dependencies: 
  - icu >=73.1,<74.0a0
  - libiconv >=1.16,<2.0a0
  - xz >=5.4.6,<6.0a0
  - zlib >=1.2.13,<1.3.0a0
```

```
$CONDA_SUBDIR=linux-64 conda search libxml2=2.13.5 --info
...
dependencies: 
  - icu >=73.1,<74.0a0
  - libgcc-ng >=11.2.0
  - xz >=5.4.6,<6.0a0
  - zlib >=1.2.13,<1.3.0a0
```

Reply:
Why `libxml2` does not pull in its "own" `libiconv` then?
https://github.com/AnacondaRecipes/libxml2-feedstock/blob/master/recipe/meta.yaml

Reply:
@cbouss and I have noticed some issues with `libiconv`'s `run_exports` on osx and win sometimes. Should we update `libiconv` to **1.17** in the next sprints?